Makefile

```diff
@@ -99,11 +99,16 @@ ifeq ($(DISABLE_CGO), 1)
 	override BUILDTAGS = exclude_graphdriver_btrfs containers_image_openpgp
 endif
 
+BIN=bin/skopeo
```

Comment:
Naming this something a bit more explicit would help. `SKOPEO_BIN`? `SKOPEO_TARGET`?

---

There are quite a few other instances of `bin/skopeo` in the Makefile, why don’t they need updating?